skt/reporter.py

```diff
@@ -273,29 +273,26 @@ def stateconfigdata(self, mergedata):
         return mergedata
 
     def update_mergedata(self):
+        """Get information about the patch(es) being merged.
+
+        Returns: A dictionary of information about the merge, including the
+                 patch(es) being merged, and which git ref they are applied to.
+
+        """
         mergedata = {
             'base': None,
             'merge_git': [],
             'localpatch': [],
             'patchwork': [],
-            'config': None
         }
 
         if self.cfg.get("infourl"):
```

Comment:
I feel it's the same story with the "info file". I.e. we don't really need to download it. However, let's deal with this in a separate PR.